### ISSUE TITLE
sortInit not working when columns with display xls

### ISSUE THREAD

**Issue by rnowif** [dandelion/dandelion-datatables#282]
Hello,

When I mix xls displayed columns and html displayed columns, the sortInit attribute does not work.

```jsp
<datatables:table id="myTableId" ...>
   <datatables:export type="xls" autoSize="true" cssClass="xlsLink" label="" />
   <datatables:column titleKey="firstName" display="xls">
   </datatables:column>
   <datatables:column titleKey="lastName" display="html" sortInit="asc" sortable="true">
   </datatables:column>
   <datatables:column titleKey="country" display="html" sortable="true">
   </datatables:column>
</datatables:table>
```

I expect the table to be initially sorted by lastName, but the datatable is sorted by country instead. I think it uses the index of the column to sort. The mix of xls and html columns seems to confuse the calculation.

Thank you for your time.

**Comment by tduchateau** [comment]
Thanks for reporting Renaud, I'll look into this

**Comment by tduchateau** [comment]
This has been fixed in the v1.0.0.
